prow/bump.sh

```diff
@@ -18,19 +18,19 @@ set -o errexit
 # See https://misc.flogisoft.com/bash/tip_colors_and_formatting
 
 color-image() {  # Bold blue
-  echo -e "\e[1;35m${@}\e[0m"
+  echo -e "\x1B[1;35m${@}\x1B[0m"
```

Comment:
this is hex for the esc char instead of the \e variant which darwin's ancient bash doesn't support
cc @ixdy @stevekuznetsov :joy:

Reply:
This is why I suggested to use `tput` in the original PR :D

Reply:
FWIW I did originally do the push from linux, but the gcr push is borked on that machine :( @krzyzacy